server/src/internalClusterTest/java/org/opensearch/search/suggest/CompletionSuggestSearchIT.java

```diff
@@ -1195,10 +1199,12 @@ public void testSkipDuplicates() throws Exception {
         indexRandom(true, indexRequestBuilders);
```

Comment:
Interesting, I would have expected the refresh to be taken care of by `indexRandom` here. Regardless the additional logs should help in case the issue persists though. Thanks!

Reply:
Thanks for taking a look